subsys/bluetooth/controller/ll_sw/ull_conn_iso_types.h

```diff
@@ -8,6 +8,10 @@ struct ll_conn;
 
 typedef void (*ll_iso_stream_released_cb_t)(struct ll_conn *conn);
 
+#define CIG_STATE_IDLE         0
+#define CIG_STATE_CONFIGURABLE CIG_STATE_IDLE /* Central only */
+#define CIG_STATE_ACTIVE       1
+#define CIG_STATE_INACTIVE     2
```

Comment:
Have unique values

```suggestion
#define CIG_STATE_IDLE         0
#define CIG_STATE_CONFIGURABLE 1 /* Central only */
#define CIG_STATE_ACTIVE       2
#define CIG_STATE_INACTIVE     3
```

Reply:
`IDLE` is not a state per the spec, are you sure we want a separate `IDLE` state?

Reply:
Then how about one `CIG_STATE_NO_CIG_OR_CONFIGURABLE`; and have `CIG_STATE_ACTIVE` and `CIG_STATE_INACTIVE`

Reply:
Better, but still, to me the initial state of a CIG which has been alloc'ed into existence, and gets the state 'CIG_STATE_NO_CIG_XXX' seems a bit odd ;-) How about  `CIG_STATE_IDLE_OR_CONFIGURABLE`?

Reply:
`NO CIG`  is the BT spec defined term in the state machine, hence I will stick with `CIG_STATE_NO_CIG_OR_CONFIGURABLE` or have separate `CIG_STATE_NO_CIG` and `CIG_STATE_CONFIGURABLE`. I vote for the latter, separate states (like the Spec uses in the fig 4.65).